doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1277,7 +1325,14 @@ constructions.
 
 .. _Fixpoint:
 
-.. cmd:: Fixpoint @ident @binders {? {struct @ident} } {? : @type } := @term
+.. cmd:: Fixpoint {+with @fix_definition }
+
+   .. insertprodn fix_definition decl_notation
+
+   .. prodn::
+      fix_definition ::= @ident_decl {* @binder } {? @fixannot } {? : @term } {? := @term } {? @decl_notations }
```

Comment:
Interesting! The following is syntactically correct:

```
Fixpoint foo.
```

However, it raises an error nonetheless: "A fixpoint needs at least one parameter".

This contrasts with the grammar of `Definition` where the choice that was made was to eliminate a definition without body nor type at parsing time.